### PR TITLE
Bug 1873125: OpenStack:  UPI missing instructions to update apiVIP and ingressVIP

### PR DESCRIPTION
**Pull request by adduarte**
Added required documentation describing the roles
and requirements of apiVIP and ingressVIP fields in install-config.yaml

### CODE REVIEW

File: docs/user/openstack/install_upi.md [hunk 1]
```diff
@@ -264,16 +264,33 @@ $ tree
...
+### Configure the machineNetwork CIDR, apiVIP, and ingressVIP.
+The machineNetwork represents the OpenStack network which will be used to connect all the OpenShift nodes that make up the
+cluster (master, workers, etc...). The machineNetwork CIDR must match the os_subnet_range CIDR specified in the inventory.yaml file
+so the installer will know what ip addresses to use for the nodes. 
+In the previous step, the installer added default values for the machineNetwork CIDR, and then it picked two default ip addresses from the
+range specified by the CIDR to assign to the the ingress and api VIPs. If you want to use your own CIDR, you have to change
+these default values in the install-config.yaml file.
 
-The installer added a default IP range for the OpenShift nodes. It must match the range for the Neutron subnet you'll create later on.
+If you change the machineNetwork CIDR but you don't chang the apiVIP and ingressVIP, the installer will error out when
```
Comment (luis5tb) [inline docs/user/openstack/install_upi.md:275]: typo: you don't change

File: docs/user/openstack/install_upi.md [hunk 2]
```diff
@@ -264,16 +264,33 @@ $ tree
 └── install-config.yaml
 ```
 
-### Fix the Node Subnet
+### Configure the machineNetwork CIDR, apiVIP, and ingressVIP.
+The machineNetwork represents the OpenStack network which will be used to connect all the OpenShift nodes that make up the
```
Comment (mandre) [inline docs/user/openstack/install_upi.md:268]: Filenames and attributes should be enclosed with backticks: `machineNetwork`, `os_subnet_range`, `inventory.yaml`, etc.

File: docs/user/openstack/install_upi.md [hunk 3]
```diff
@@ -264,16 +264,33 @@ $ tree
...
+cluster (master, workers, etc...). The machineNetwork CIDR must match the os_subnet_range CIDR specified in the inventory.yaml file
+so the installer will know what ip addresses to use for the nodes.
```
Comment (mandre) [inline docs/user/openstack/install_upi.md:270]: Can we also make sure to uppercase acronyms such as IP and API?

File: docs/user/openstack/install_upi.md [hunk 4]
```diff
@@ -264,16 +264,30 @@ $ tree
 └── install-config.yaml
 ```
 
-### Fix the Node Subnet
+### Configure the machineNetwork.CIDR apiVIP and ingressVIP.
```
Comment (mandre) [inline docs/user/openstack/install_upi.md:267]: Extra `.` at the end.

File: docs/user/openstack/install_upi.md [hunk 5]
```diff
@@ -264,16 +264,30 @@ $ tree
...
+The `machineNetwork` represents the OpenStack network which will be used to connect all the OpenShift cluster nodes.
+The `machineNetwork.CIDR` defines the IP range, in CIDR notation, from which the installer will choose what IP addresses
+to assign the nodes.  The `apiVIP` and `ingressVIP` are the IP addresses the installer will assign to the cluster API and
+ingress VIPs, respectively.
+In the previous steps, the installer added default values for the `machineNetwork.CIDR`, and then it picked the
+5th and 7th IP addresses from that range to assign to `apiVIP` and `ingressVIP`.
+`machineNetwork.CIDR` needs to match the IP range specified by `os_subnet_range` in the inventory.yaml file.
```
Comment (mandre) [inline docs/user/openstack/install_upi.md:274]: `inventory.yaml` with backticks.

File: docs/user/openstack/install_upi.md [hunk 6]
```diff
@@ -264,16 +264,30 @@ $ tree
...
+`apiVIP` and `ingressVIP` fall within the IP range specified by `machineNetwork.CIDR`. If they do not, it errors out.
+If you change the value of `machineNetwork.CIDR` you must make sure the `apiVIP` and `ingresVIP` values still fall within
+the new range. There are two options for setting the `apiVIP` and `ingressVIP`. If you know the values you want to use,
+you can specify them in the `install-config.yaml` file. If you want the installer to pick the 5th and 7th IP addresses in the
+new range, you need to remove the `apiVIP` and `ingressVIP` entries from the `install-config.yaml` file.
 
-We're going to use a custom subnet to illustrate how that can be done.
+To illustrate the process, we will use '192.0.2.0/24' as an example. It defines a usable IP range from
+192.0.2.1 to 192.0.2.254. There are some IP addresses that should be avoided because they are usually taken up or
+reserved. For example, the first address (.1) is could be assigned to a router. The DHCP and DNS servers will use a few
```
Comment (mandre) [inline docs/user/openstack/install_upi.md:285]: > the first address (.1) is could be assigned to a router

There's one extra verb that slipped in. Maybe you could say `the first address (.1) is usually assigned to a router`

Comment (adduarte) [reply docs/user/openstack/install_upi.md:285]: how that get in there???
will correct.

File: docs/user/openstack/install_upi.md (hunk 4 above)
Comment (adduarte) [reply docs/user/openstack/install_upi.md:267]: done

File: docs/user/openstack/install_upi.md (hunk 5 above)
Comment (adduarte) [reply docs/user/openstack/install_upi.md:274]: done

**Pull request merged**